packages/core/lib/src/internal/ops/text_style.dart

```diff
@@ -64,7 +64,7 @@ class TextStyleOps {
         ),
       );
 
-  static TextStyleHtml Function(TextStyleHtml, String) fontSize(
+  static TextStyleHtml Function(TextStyleHtml, String?) fontSize(
```

Comment:
Is it possible to change all of these into non-nullable `String`?

Reply:
Yep, changed.